### PR TITLE
[seti][boinc] change seti.pl  with boinc.pl

### PR DESCRIPTION
**Pull request by bw1**

### CODE REVIEW

File: scripts/boinc.pl [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+	'enable' => sub {$a_disable =0},
+	'list' => \&cmd_clist,
+);
+
+my $boinc_cmd='boinccmd';
+my $boinc_cmd_host='--host';
+my $boinc_cmd_passwd='--passwd';
+my $boinc_cmd_state='--get_state';
+my $boinc_cmd_old_task='--get_old_tasks';
+my @hosts= qw/rpi1.my rpi2.my rpi3.my rpi4.my/;
```
Comment (ailin-nemui) [inline scripts/boinc.pl]: is this a useless variable?

Comment (bw1) [reply scripts/boinc.pl]: :| 
a artefact

File: scripts/boinc.pl [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+	$readex{$pid}->{out}=$out;
+	$readex{$pid}->{err}=$err;
+	$readex{$pid}->{host}=$host;
+	$readex{$pid}->{rfunc}=$rfunc;
+
+	Irssi::pidwait_add($pid);
+}
+
+sub sig_read_exec {
+	my ($pid, $status) = @_;
```
Comment (ailin-nemui) [inline scripts/boinc.pl]: you should check if this is your PID, and not from some other script

**Comment by bw1** [comment]
done

**Pull request merged**